pom.xml

```diff
@@ -56,6 +56,66 @@
 			<artifactId>spring-boot-starter-test</artifactId>
 			<scope>test</scope>
 		</dependency>
+		<dependency>
```

Comment:
Please justify the need for each of the new dependencies. (Make sure to also remove blank lines to keep the code clean)

Reply:
io.jsonwebtoken needed for JWT Token generation and manipulation

`<dependency>
	<groupId>org.springframework.boot</groupId>
	<artifactId>spring-boot-starter-security</artifactId>
</dependency>`
Needed for Spring Security and filters


<dependency>
	<groupId>com.itextpdf</groupId>
	<artifactId>itextpdf</artifactId>
	<version>5.5.13.3</version>
</dependency>
Needed further ahead for printing labels


<dependency>
	<groupId>com.fasterxml.jackson.core</groupId>
	<artifactId>jackson-databind</artifactId>
	<version>2.14.2</version>
</dependency>
Jackson needed by the JWT library for claim type conversion.

Please let me know if there is any additional explanation needed for these

Reply:
Remove com.itextpdf itextpdf 5.5.13.3. You've justified 4 dependencies but added 8. Please justify ALL